content/en/continuous_integration/setup_tests/junit_upload.md

```diff
@@ -30,6 +30,20 @@ Install the [`datadog-ci`][2] CLI globally using `npm`:
 npm install -g @datadog/datadog-ci
 {{< /code-block >}}
 
+### Standalone binary (beta)
+
+If installing nodejs in the CI is an issue, a standalone binary is provided with releases. Only _linux-x64_, _darwin-x64_ (macOS) and _win-x64_ (Windows) are supported at the time. To install for linux:
```

Comment:
I would actually apply the architecture nomenclature you're using here to the download filenames:

- datadog-ci-linux -> datadog-ci_linux-x64
- datadog-ci-win -> datadog-ci_win-x64
- datadog-ci-macos -> datadog-ci_darwin-x64